news/_posts/2016-05-10-prestashop-coding-standards.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: post
+title:  "PrestaShop Coding Standards"
+subtitle: "How to contribute to PrestaShop project"
+date:   2016-05-10 03:00:00
+author: [ maximebiloe ]
+icon: icon-code
+tags:
+- coding-standards
+- development
+---
+
+
+Working on an open source project as massive as the PrestaShop codebase, a few ground rules have long been established in order to ease the participation of developers, be they from the PrestaShop Core team or the Community at large.
+
+For long, we used our own coding standards. We then switched to PSR-2 last year.
+
+Because we want to have an always better codebase, clearer, easier to read and to maintain, we have decided to follow some more established standards for the languages used in PrestaShop.
+
+This means that we are updating our current standards requirements for languages that already have one, or selecting a standard for the languages which did not have one in use yet.
```

Comment:
This means that we are updating our current standards requirements for languages that already have some, or selecting a standard for the languages which did not have any in use yet.

(some pour standards requirements et plus one pour standard)